container/devel/README.md

```diff
@@ -0,0 +1,171 @@
+# Setup a Development Enviroment for openQA and related Tools
+
+These examples use `podman` and
+[`distrobox`](https://en.opensuse.org/Distrobox). You can use similar tools
+like `docker`, `toolbox` etc. to achieve the same.
+
+You will run two containers:
+* A separate postgres container (so you don't have to deal with the database
+  when updating your dev container)
+* The openQA environment in a openSUSE Leap or Tumbleweed container, which
+  both can use the same postgres container.
+  In this example we will be settting up and using the Leap container.
```

Comment:
I suggest to link to http://open.qa/docs/#contributing if not rather move the documentation there. Right now unfortunately it seems this documentation is a complete alternative to http://open.qa/docs/#development-setup confusing users which instructions they should follow.

Reply:
That's why it's a draft